src/interactive/Dropdown.jsx

```diff
@@ -58,7 +58,11 @@ class Dropdown extends React.PureComponent {
 
 	onKeyDown(e) {
 		if (e.key === 'Enter' && this.state.isActive) {
-			this.closeContent();
+			// postpoining closure till next cycle
+			// otherwise closure cancels default keyboard events
+			setTimeout(() => {
+				this.closeContent();
+			}, 0);
```

Comment:
Little confused, we are calling `this.closeContent` after 0 milliseconds? Is `setTimeout` really postponing if we aren't specifying a larger wait interval?

Reply:
It just calls that method async, rather than running that code right away.

Reply:
no need to wrap the this.closeContent in a function

`setTimeout(this.closeContent, 0)`